internal/uvm/virtual_device.go

```diff
@@ -31,6 +36,21 @@ type VPCIDevice struct {
 	refCount uint32
 }
 
+// GetAssignedDeviceVMBUSID returns the instance ID of the VMBUS instance created.
+//
+// A device's instance ID is an identifier that distinguishes a device from other devices
+// on the system. The instance GUID of a VMBUS instance refers to that bus's unique
+// identifier used internally by VMBUS and can be used to determine the VMBUS
+// instance's instance ID.
```

Comment:
This sounds like "we use the "VMBUS instance's instance GUID to determine the VMBUS instance's instance ID", which sounds very confusing and doesn't make any sense to me. Can you clarify the comment a bit? Is "instance" overloaded multiple times in this terminology?

It might be helpful to start with a breakdown of the different types of ID and what they refer to.

Reply:
Also an example with some real IDs might help.

Reply:
I'm not sure an example of a real ID would be helpful considering there's already a format mentioned: 

```
// A VMBUS's instance ID is in the form: "VMBUS\vmbusChannelGUIDFormatted\{vmBusInstanceGUID}"
```

Reply:
instance here refers to both an individual instance of a device as well as is part of the term "instance ID" which is used by pnp

Reply:
New iteration of this seems much better https://github.com/microsoft/hcsshim/pull/842/files#diff-725ceffd9e33933a28c9e17d77d509d9R39-R50

Reply:
This looks better, thank you.